src/main/kotlin/screeps/utils/Interop.kt

```diff
@@ -26,7 +26,7 @@ package screeps.utils
  *     }
  *
  */
-fun <T> jsObject(block: T.() -> Unit): T = Any().unsafeCast<T>().also(block)
+fun jsObject(block: dynamic.() -> Unit): dynamic = Any().also(block)
```

Comment:
I feel like the fact that we *we* call `.unsafeCast<T>()` for an unconstrained type is bad.
What do you think about saying `fun jsObject(block: T.() -> Unit): dynamic = Any().also(block)`? You get the autocomplete from `T`, but you still have to cast it explicitly.

Reply:
I am not sure what we would gain by that.
There is already a KDoc comment explaining the risks of using this method. It might be appropriate to rename the package for this file to screeps.utils.unsafe since that is what both methods are